examples/annotation_label_studio/materializers/pillow_image_materializer.py

```diff
@@ -0,0 +1,45 @@
+import os
+import tempfile
+from typing import Dict, Type
+
+from PIL import Image
+
+from zenml.artifacts import DataArtifact
+from zenml.io import fileio
+from zenml.logger import get_logger
+from zenml.materializers.base_materializer import BaseMaterializer
+from zenml.utils import io_utils
+
+logger = get_logger(__name__)
+
+
+class PillowImageMaterializer(BaseMaterializer):
```

Comment:
So this materializer takes a dict of files and returns a dict of images? Perhaps we should make that clear in the docstrings as it might be confusing

Reply:
Done